Kickstarter-iOS/Views/Controllers/ProjectPamphletContentViewController.swift

```diff
@@ -151,7 +151,7 @@ public final class ProjectPamphletContentViewController: UITableViewController {
 
   private func goToLiveStream(project: Project, liveStream: Project.LiveStream) {
     let vc: UIViewController
-    if liveStream.startDate < Date().timeIntervalSince1970 {
+    if liveStream.isLiveNow || liveStream.startDate < Date().timeIntervalSince1970 {
```

Comment:
Think I'm going to make this two separate outputs rather so we can test it (goToLiveStream and goToLiveStreamCountdown) 👍